docker/npm-build-pod/Dockerfile

```diff
@@ -8,8 +8,8 @@ RUN yum remove -y rh-nodejs4; \
       --enablerepo=rhel-server-rhscl-7-rpms \
       --enablerepo=rhel-7-server-optional-rpms \
       --enablerepo=rhel-7-server-optional-rpms \
-      --disablerepo=epel
       $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
-    yum clean all -y
+    yum clean all -y \
+    rm -rf /var/cache/yum
```

Comment:
What is the purpose for force removing yum cache dir after issuing yum clean all -y?

Reply:
When you run `yum clean all -y` it actually prints a message suggesting that you ALSO do the `rm -rf /var/cache/yum`